src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -385,4 +404,34 @@ public function orderShouldNotExistInTheRegistry(OrderInterface $order)
             sprintf('Order with number %s exists but should not.', $order->getNumber())
         );
     }
+
+    /**
+     * @Then I should be notified that the order's payment has been successfully completed
```

Comment:
I'd leave this step definition, later we will change the flash (configurable in routing, yay) for better UX.